Contributing.md

```diff
@@ -77,6 +77,22 @@ git push origin YourBranchName
 
 **_:trophy: After this, the maintainers will review the PR and will merge it if it helps move the LinkFree project forward. Otherwise, it will be given constructive feedback and suggestions for the changes needed to add the PR to the codebase._**
 
+## Run automated tests
```

Comment:
What if we add an emoji here? Would that be good?

Reply:
Yes, emojis are important for the document on how to contribute, otherwise, users will have a hard time reading it. What type of emoji are you talking about?

Reply:
Emojis like this https://github.com/EddieHubCommunity/LinkFree#-demo
Do you think it'd look good? Let me know what do you think. 🙂

Reply:
If you think it's worth adding, please create another PR. This is not related to the issue I'm fixing. Which is missing documentation of running tests locally.

Reply:
Well, you're actually adding the documentation, so I thought of adding it on this PR as well 😅 But if you want, let's keep it as-is then.

Reply:
I can change the heading to `###` since this should probably be part of the PR section, the level 3 subheading don't have emoji.